---
layout: default
title: Wang Lab
subtitle: Chenfei Wang Lab at Tongji University
use-site-title: true
---

<div class="container">
    <div class="row">
        <div id="carouselExampleIndicators" class="carousel slide carousel-fade" data-ride="carousel">
          <!-- <ol class="carousel-indicators">
            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
            <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
          </ol> -->
          <div class="carousel-inner">
            <div class="carousel-item active" style="height: 25rem">
              <img src="/assets/img/lab_2024.jpg" class="d-block w-100">
            </div>
<!--             <div class="carousel-item" style="height: 25rem">
              <img src="/assets/img/lab.jpg" class="d-block w-100">
            </div> -->
          </div>
          <!-- <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a> -->
        </div>
    </div>

    <div class="row" style="margin-top: 3rem">
        <h2>Research interests</h2>
        <p>Our research is dedicated to developing computational algorithms and conducting large-scale data mining from single-cell multiomic data. We aim to unravel the intricate mechanisms of gene regulation, cellular crosstalk, and metabolic reprogramming that define cell identities. Additionally, we have created methods and web resources specifically designed for analyzing single-cell multi-omics, spatial multi-omics, single-cell CRISPR screen data, performing signal enhancement, and multimodal translation. By seamlessly integrating single-cell datasets with spatial and multimodal information, our goal is to establish quantitative connections between changes in cell identity and disease phenotypes in various biological systems, including cancer, development, and aging.
        </p>
    </div>
    <div class="row">
        <h3>Intelligent Algorithms for Enhancing, Generating, and Functionally Analyzing Single-Cell Spatial Multi-Omics</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/STRIDE_Cellist.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Single-cell and spatial multi-omic technologies have revolutionized our understanding of cellular heterogeneity in complex biological systems. However, corresponding analyses currently face various challenges, including inadequate resolution, coverage, and difficulty integrating and generating heterogeneous multi-modal data. To address these issues, we have developed a series of intelligent algorithms. <a href="https://github.com/wanglabtongji/STRIDE">STRIDE</a> and <a href="https://github.com/wanglabtongji/Cellist">Cellist</a> enhance low-resolution and high-resolution spatial transcriptomics data signals, respectively, elevating them to the single-cell level. <a href="https://github.com/wanglabtongji/SCRIP">SCRIP</a> and <a href="https://github.com/wanglabtongji/SCRIPro">SCRIPro</a> constructs gene regulation networks using single-cell and spatial multiomics data. <a href="https://github.com/wanglabtongji/CCI">EvaCCI</a> assesses intercellular interactions , while <a href="https://github.com/wanglabtongji/SCREE">SCREE</a> analyzes multimodal single-cell CRISPR screen data. These algorithms enhance the usability of single-cell spatial omics data and lay the foundation for the integrative analysis of spatiotemporal multimodal data, as well as computational modeling of cellular phenotypes in physiological and pathological conditions.
                </p>
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Cell Phenotype Modeling Based on Generative AI and Large-Scale Spatiotemporal Multimodal Data</h3>
        <div class="row">
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cell phenotypes in multicellular systems are regulated by intrinsic factors (such as gene expression regulation) and extrinsic factors (such as intercellular interactions). We have demonstrated the tight connection between intrinsic epigenetic regulations and cell fate determination in mouse IVF and SCNT embryo development (<b>Nature, 2016; Nat. Cell Biol., 2018; Cell Stem Cell, 2018, 2022; Cell Res., 2022</b>). Currently, we are developing generative AI models that are pretrained on large-scale single-cell multi-modal datasets. These models aim to uncover the collaborative effects between gene regulation, cellular crosstalk, and other environmental factors, such as metabolites and mechanics, on cell phenotypes. As a pioneering work, we have developed <a href="https://github.com/wanglabtongji/SELINA.py">SELINA</a>, which utilizes a multi-adversarial domain adaptation network to automatically annotate cell types using a large-scale pretrained human scRNA-seq reference. We hope to leverage generative AI models to gain deeper insights into the molecular mechanisms driving cell phenotypes and further guide and reshape this transformation process.
                </p>
            </div>
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/SELINA.png">
            </div>
        </div>
    </div>
    <div class="row">
        <h3>Diversity, Plasticity, and Association of Cell Phenotypes to Individual Phenotypes in Complex Disease Microenvironments</h3>
        <div class="row">
            <div class="col-lg-4">
                <img style="margin-top: 1.5rem;" src="/assets/img/research/TabulaTIME.png">
            </div>
            <div class="col-lg-8">
                <p style="margin-top: 1.5rem;">Cancer results from immune surveillance evasion, with the immunosuppressive tumor microenvironment having a profound impact on tumor development and treatment resistance. Our research team combines deep bioinformatics data analysis with single-cell spatiotemporal multi-omics data to study the effects of intrinsic gene expression regulation and extrinsic environmental factors on disease-related cell phenotypes in the tumor microenvironment, as well as the association between cell phenotypes and individual phenotypes. We have established a comprehensive single-cell RNA sequencing data resource, <a href="http://tisch.comp-genomics.org">TISCH</a>, for analyzing gene expression and cellular composition in the tumor microenvironment. We have also constructed a cell phenotype atlas for all cancer types,  <a href="http://timer2.compbio.cn/TabulaTIME">TabulaTIME</a> ), and discovered a widely prevalent profibrotic ecotype that regulates tumor immunity. Our team is currently collaborating closely with oncologists and immunologists to investigate the mechanisms of tumor microenvironment evolution and immune therapy resistance in different cancer types (<b>Cell 2024; Nat. Genet. 2024; Cancer Immunol. Res. 2023; Genome Med. 2023; EMBO J. 2023</b>).
                </p>
            </div>
        </div>
    </div>
  </div>
</div>


